pyuvdata/uvcal/fhd_cal.py

```diff
@@ -116,6 +117,34 @@ def read_fhd_cal(
         longitude = np.deg2rad(float(obs_data["LON"][0]))
         altitude = float(obs_data["ALT"][0])
 
+        # This is a bit of a kludge because nothing like a phase center name
+        # exists in FHD files.
+        # At least for the MWA, obs.ORIG_PHASERA and obs.ORIG_PHASEDEC specify
+        # the field the telescope was nominally pointing at
+        # (May need to be revisited, but probably isn't too important)
+        cat_name = (
+            "Field RA(deg): "
+            + str(obs_data["ORIG_PHASERA"][0])
+            + ", Dec:"
+            + str(obs_data["ORIG_PHASEDEC"][0])
+        )
+        # For the MWA, this can sometimes be converted to EoR fields
+        if self.telescope_name.lower() == "mwa":
+            if np.isclose(obs_data["ORIG_PHASERA"][0], 0) and np.isclose(
+                obs_data["ORIG_PHASEDEC"][0], -27
+            ):
+                cat_name = "EoR 0 Field"
+
+        cat_id = self._add_phase_center(
+            cat_name=cat_name,
+            cat_type="sidereal",
+            cat_lon=np.deg2rad(float(obs_data["OBSRA"][0])),
+            cat_lat=np.deg2rad(float(obs_data["OBSDEC"][0])),
+            cat_frame=astrometry["RADECSYS"][0].decode().lower(),
+            cat_epoch=astrometry["EQUINOX"][0],
```

Comment:
It's an optional parameter, but I usually try to add `info_source="file"` or similar here, just so that the origin of the information is clear (i.e., not something that the user created)